MakeWin.bat

```diff
@@ -0,0 +1,150 @@
+if %ERRORLEVEL% neq 0  call %0 pip
+REM Check if numpy is installed
+python -c "import numpy" 2>nul
+if %ERRORLEVEL% neq 0  call %0 numpy
+
+REM Retrieve and install simphony library
+pip install --upgrade git+https://github.com/simphony/simphony-common.git@%SIMPHONYVERSION%#egg=simphony
+goto exit
+
+
+:simphony-ncad
+echo ----- INSTALLING SIMPHONY-NCAD -----
+REM Check if pip is installed
+python -c "import pip" 2>nul
+if %ERRORLEVEL% neq 0  call %0 pip
+REM Check if cython is installed
+python -c "import cython" 2>nul
+if %ERRORLEVEL% neq 0  pip install cython
+REM Check if simphony is installed
+python -c "import simphony" 2>nul
+if %ERRORLEVEL% neq 0  call %0 simphony
+
+pip install --upgrade git+https://github.com/simphony/simphony-nCAD.git@%NCADVERSION%#egg=simphony_ncad
+goto exit
+
+
+:nfluid
+echo ----- INSTALLING NFLUID -----
+REM Check if pip is installed
+python -c "import pip" 2>nul
+if %ERRORLEVEL% neq 0  call %0 pip
+REM Check if numpy is installed
+python -c "import numpy" 2>nul
+if %ERRORLEVEL% neq 0  call %0 numpy
+REM Check if simphony is installed
+python -c "import simphony" 2>nul
+if %ERRORLEVEL% neq 0  call %0 simphony
+
+pip install --upgrade git+https://github.com/simphony/nfluid.git@%NFLUIDVERSION%#egg=nfluid
+goto exit
+
+
+:nfluid-dev
+echo ----- INSTALLING NFLUID-DEV -----
+pip install --upgrade git+https://github.com/simphony/nfluid.git@nfluid-goyo-dev#egg=nfluid-goyo-dev
+goto exit
+
+
+:clean
+echo ----- REMOVING SIMPHONY, SIMPHONY-NCAD AND NFLUID -----
+for /d %%f in ("C:\Python27\Lib\site-packages\ncad_wrapper*") do rd /s /q "%%~f"
+for /d %%f in ("C:\Python27\Lib\site-packages\nfluid*") do rd /s /q "%%~f"
+for /d %%f in ("C:\Python27\Lib\site-packages\simncad*") do rd /s /q "%%~f"
+for /d %%f in ("C:\Python27\Lib\site-packages\simphony*") do rd /s /q "%%~f"
+goto exit
+
+
+:pip
+REM Get pip package
+REM -- using urllib
+python -c "import urllib; urllib.urlretrieve('https://bootstrap.pypa.io/get-pip.py', 'C:\python27\Tools\get-pip.py')"
+REM -- using urllib2
+REM python -c "import urllib2; ifile=urllib2.urlopen('https://bootstrap.pypa.io/get-pip.py');ofile=open('C:/python27/Tools/pip.py','wb');ofile.write(ifile.read());ofile.close()"
+REM Install pip
+python C:\python27\Tools\get-pip.py
+goto exit
+
+:numpy
+REM Get unofficial numpy (installation of official package is not successful on Windows)
+REM from http://www.lfd.uci.edu/~gohlke/pythonlibs/
+REM -- using urllib2 with proper User-Agent
+python -c "import urllib2; headers={'User-Agent':'Mozilla/5.0 (Windows; U; MSIE 7.0; Windows NT 6.0; en-US)'}; request=urllib2.Request('http://www.lfd.uci.edu/~gohlke/pythonlibs/wkvprmqy/numpy-1.10.4+vanilla-cp27-none-win32.whl',None,headers); ifile=urllib2.urlopen(request); ofile=open('C:/Python27/Lib/site-packages/numpy-1.10.4+vanilla-cp27-none-win32.whl','wb'); ofile.write(ifile.read()); ofile.close()"
+REM Install unofficial numpy
+pip install C:\Python27\Lib\site-packages\numpy-1.10.4+vanilla-cp27-none-win32.whl
```

Comment:
This is just a temporary solution because of current problems to install the _official_ `numpy` version. I guess that future `numpy` releases will be able to compile on Windows as well.